Cookbook/Technical-Documents/Development-Process.md

```diff
@@ -0,0 +1,63 @@
+# Development Process
+
+At Babylon security is something we have always in mind when developing software and that is why we follow the SSDLC (Secure Software Development Life Cycle) framework.
+Alongside this framework, we also use a few services to help us keep track of what we are building and make sure we deliver a secure and stable product.
+
+
+There are 3 services we use daily:
+
+- Jira: is where the development process begins. Once a ticket has all the necessary details, dev can pick it up and implement it <!-- TODO: [CNSMR-3230] link to Jira article -->
+- GitHub: used for hosting our project and version control using Git
+- CircleCI: our continuous integration service to run tests when integrating PRs <!-- TODO: [CNSMR-xxxx] link to CircleCI article -->
+
+## 1. Get a Jira ticket 🎫
+
+- Every development task that results in a PR must have a Jira ticket
+- The ticket must be assigned to the engineer who is going to work on it. The engineer should make sure it has all details and designs attached (if applicable).
+- The ticket will be moved to `In progress` once development is ready to start (this might be automatically done by the bot when branch published, if configured)
```

Comment:
```suggestion
- The ticket will be moved to `In progress` once development is ready to start (this might be automatically done by the bot when branch is published, if configured)
```